.github/deploy/steps/65-Provision-Spn-db-user.ps1

```diff
@@ -10,7 +10,8 @@ Graph-CreateRole -principalId $spId -roleDefinitionId 88d8e3e3-8f55-4a1e-953a-9b
 Ignore-Errors  # The role may already be there if we re-deploy
 
 Write-Host "Waiting for role to settle...." -ForegroundColor DarkYellow
-Start-Sleep -seconds 60 # If the Directory Reader is not there, increase te seconds here.
+#Start-Sleep -seconds 60 # If the Directory Reader is not there, increase te seconds here.
+#skip this and let's see where it fails
```

Comment:
Azure AD is a bit slow sometimes. The SQL server identity needed "Directory Reader" role. That takes apparently some time before AD gives the role.